site/docs/output_directories.md

```diff
@@ -90,36 +90,42 @@ The directories are laid out as follows:
       server/                             <== The Bazel server puts all server-related files (such as socket
                                               file, logs, etc) here.
         jvm.out                           <== The debugging output for the server.
-      &lt;workspace-name&gt;/             <== Working tree for the Bazel build & root of symlink forest: execRoot
-        _bin/                             <== Helper tools are linked from or copied to here.
-
-        bazel-out/                        <== All actual output of the build is under here: outputPath
-          local_linux-fastbuild/          <== one subdirectory per unique target BuildConfiguration instance;
+      execroot/                           <== The working directory for all actions. For special cases such
```

Comment:
> The working directory for all actions. For special cases such as sandboxing and remote execution, the actions run in a directory that mimics execroot.

Please end the sentence here and begin a new one at "Implementation details". It was difficult for me to parse the sentence in its current form.